semgrep/semgrep/error.py

```diff
@@ -253,7 +256,13 @@ def __str__(self) -> str:
             help_str = f"= {with_color(Fore.CYAN, 'help', bold=True)}: {self.help}"
         else:
             help_str = ""
-        return f"{header}\n{snippet_str}\n{help_str}\n{with_color(Fore.RED, self.long_msg or '')}\n"
+
+        # TODO remove this when temp files are no longer in error messages
+        if snippet_str == "":
+            snippet_str_with_newline = ""
+        else:
+            snippet_str_with_newline = f"{snippet_str}\n"
```

Comment:
In Python you can write: `snippet_str_with_newline = f"{snippet_str}\n" if snippet_str != "" else ""`